docs/data/sdks/flutter/index.md

```diff
@@ -59,7 +59,44 @@ class YourClass {
 }
 ```
 
-### EU data residency
+### Configuration
+
+Amplitude Flutter SDK runs on the top of the [Amplitude Android Mantaince SDK](../android/), [Amplitude iOS Mantaince SDK](../ios/) and [Amplitude JavaScript Mantaince SDK](../javascript/). The following are the Dart settable config options.
+For more configurations:
+
+- on Android side, check the [Android Configuration](../android-kotlin/#configuration)
```

Comment:
As we mention the configuration on other SDKs, I guess there is no good way in flutter to call these configurations? Maybe we can say for other default configurations or default behavior, they could refer there.